src/frida/repl.py

```diff
@@ -423,6 +431,57 @@ def _create_repl_script(self):
                 with codecs.open(self._user_script, 'rb', 'utf-8') as f:
                     user_script = f.read().rstrip("\r\n") + "\n\n// Frida REPL script:\n"
 
+            if self._codeshare_uri is not None:
+                trust_store = self._get_or_create_truststore()
+                project_url = 'https://codeshare.frida.re/api/project/{}/'.format(self._codeshare_uri)
```

Comment:
We should stick to double quotation marks and only use single quotation marks for symbols/atoms (enum values). In hindsight this is probably too weird and something we might want to move away from at some point (in one big swoop), but in the meantime it's good to be consistent.